Mathlib/Data/Int/Cast/Lemmas.lean

```diff
@@ -205,9 +205,11 @@ theorem cast_natAbs : (n.natAbs : α) = |n| := by
 
 end LinearOrderedRing
 
-theorem coe_int_dvd [CommRing α] (m n : ℤ) (h : m ∣ n) : (m : α) ∣ (n : α) :=
+theorem cast_dvd_cast [CommRing α] (m n : ℤ) (h : m ∣ n) : (m : α) ∣ (n : α) :=
```

Comment:
```suggestion
theorem intCast_dvd_intCast [CommRing α] (m n : ℤ) (h : m ∣ n) : (m : α) ∣ (n : α) :=
```
for coherence?

Reply:
It's in namespace `Int` unless I've messed up.

Reply:
Indeed, I just checked and it is in namespace `Int`